packages/suite/src/components/wallet/BigAmountValue.tsx

```diff
@@ -47,7 +47,7 @@ export const BigAmountValue = ({
     const language = useSelector(selectLanguage);
 
     // Todo: this is ugly hack, shall be refactored to some more safe alternative
-    const [whole, separator, fractional] = ['en', 'ja', 'zh'].includes(language)
+    const [whole, separator, fractional] = ['en-US', 'ja-JP', 'zh-CN', 'zh-TW'].includes(language)
```

Comment:
We have already a list of the languages we support with their locales (eg `en-US`), so, in this case 

Would it be then possible to have this array typed by this array of locales, so that TS will tell that we are comparing randomg string to a constat list / keys?